### PR TITLE
Fix chat ordering

### PR DESCRIPTION
**Pull request by deadbender**
This PR (in local testing) has proven to fix the issues caused through removing the BOOTSTRAP action grab of the chat data. I basically just copied over what we had on the profile page that is working, and it seems to work here and now creates only 1 request on /chats.

### CODE REVIEW

**Review by JnBrymn — APPROVED** [review]
LGTM

File: penny_university_frontend/src/pages/Chats.tsx [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
 
-const ChatsPage = () => (
-  <ChatList filter={{ key: 'all', query: '' }} />
-)
+type DispatchProps = {
+  loadChatsList: (nextPageUrl?: string, userID?: string) => void,
+}
 
-export default ChatsPage
+type TParams = { id: string };
```
Comment (nchouard) [inline penny_university_frontend/src/pages/Chats.tsx:14]: Why is this here?

File: penny_university_frontend/src/pages/Chats.tsx [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+import { RouteComponentProps } from 'react-router-dom'
+import ChatList from '../components/chats/ChatList'
+import { loadChatsList } from '../actions/chat'
+import ApiRoutes from '../constants'
 
-const ChatsPage = () => (
-  <ChatList filter={{ key: 'all', query: '' }} />
-)
+type DispatchProps = {
+  loadChatsList: (nextPageUrl?: string, userID?: string) => void,
```
Comment (nchouard) [inline penny_university_frontend/src/pages/Chats.tsx:11]: Do we need userID here?

File: penny_university_frontend/src/pages/Chats.tsx [hunk 3]
```diff
@@ -1,8 +1,36 @@
...
+  }, [loadChatsList])
+  return (
+    <div>
+      <ChatList filter={{ key: 'all', query: '' }} />
+    </div>
+  )
+}
+
+const mapDispatchToProps = (dispatch: ThunkDispatch<unknown, unknown, AnyAction>) => ({
+  loadChatsList: (nextPageUrl: string, userID: string) => dispatch(loadChatsList(nextPageUrl, userID)),
```
Comment (nchouard) [inline penny_university_frontend/src/pages/Chats.tsx:32]: Do we need to pass userID here?

**Review by nchouard — COMMENTED** [review]
I think you have some unnecessary props/parameters you should remove
